### DISCUSSION TITLE
Android app now working better than ever

### DISCUSSION THREAD

**Discussion by jagjordi** [immich-app/immich#4454] [Show and tell]
With recent improvements on timeline build time now the Android app hasfully reached the Google photos app level in terms of viewing images.
Before that it was slow to load so I was using the browser version instead.

Congrats and thanks to the devs for the great and fast advancements on this project.

**Comment by alextran1502** [comment]
Thank you for the acknowledgment, and thanks @fyfrey and @shalong-tanwen for the contribution to the mobile app in the last couple of releases to make it better than ever.
